backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -464,6 +464,11 @@ public TopLevelDashboardMetrics getTopLevelDashboardMetrics(
       UUID facilityId, Date startDate, Date endDate) {
     Set<UUID> facilityIds;
 
+    if (startDate == null || endDate == null) {
+      // if null dates somehow get through, just return zeroes
+      return new TopLevelDashboardMetrics(0L, 0L);
```

Comment:
Maybe we should consider something here to let us monitor how often this happens.  Doesn't have to be right now, though.